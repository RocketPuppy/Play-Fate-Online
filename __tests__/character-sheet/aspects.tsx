import * as React from "react";
import * as renderer from "react-test-renderer";
import Aspects from "../../src/character-sheet/aspects";

test("renders all the pieces", () => {
  const render = renderer.create(<Aspects />);

  expect(render.toJSON()).toMatchSnapshot();
});
