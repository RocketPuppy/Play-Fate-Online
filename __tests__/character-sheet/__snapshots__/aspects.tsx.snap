// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders all the pieces 1`] = `
<div>
  <h2>
    Aspects
  </h2>
  <div>
    <label>
      <span>
        High Concept
      </span>
      <input
        onChange={[Function]}
        placeholder="Disciple of the Ivory Shroud"
        type="text"
        value=""
      />
    </label>
    <label>
      <span>
        Trouble
      </span>
      <input
        onChange={[Function]}
        placeholder="The manners of a goat"
        type="text"
        value=""
      />
    </label>
    <label>
      <span>
        New Aspect
      </span>
      <input
        onChange={[Function]}
        placeholder="I owe Old Finn everything"
        type="text"
        value=""
      />
    </label>
    <button
      disabled={true}
      onClick={[Function]}
    >
      Save Aspect
    </button>
  </div>
</div>
`;
